Dockerfile

```diff
@@ -16,11 +16,11 @@ COPY pkg/    pkg/
 COPY cmd/    cmd/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager github.com/pusher/wave/cmd/manager
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o wave github.com/pusher/wave/cmd/manager
```

Comment:
Can we add the `VERSION` thing to this? Do we have that in Wave yet?

Reply:
Ask and ye shall receive #48.

Reply:
You're always one step ahead aren't you? 🤔 😉